Community/iot_device_registration/README.md

```diff
@@ -0,0 +1,78 @@
+IoT Device Registration
```

Comment:
Hi, we need the standard copyright in the readme.

Reply:
Hi Xiao, I have added the standard copyright to the README.